dashboard/src/services/encryption.js

```diff
@@ -229,7 +229,7 @@ export const decryptItem = async (item, { decryptDeleted = false, type = "" } =
         draggable: false,
       }
     );
-    capture(`ERROR DECRYPTING ITEM : ${errorDecrypt}`, {
+    capture(`ERROR DECRYPTING ${type} ${item?._id} : ${errorDecrypt}`, {
```

Comment:
```suggestion
    capture(`ERROR DECRYPTING ${type || "ITEM"} ${item?._id} : ${errorDecrypt}`, {
```
(il est parfois vide)